docs/api-aggregation-setup.md

```diff
@@ -0,0 +1,183 @@
+
+# Setting up Service Catalog for API Aggregation in Kubernetes
+
+The aggregator is a server that sits in front of everything. 
+
+We need to provide a set of certificates to be provided as a
+certificate bundle to the APIService apiregistration endpoint.
+
+For development purposes, it is convenient to use the existing CA
+automatically set up by the kubernetes development environment.
+
+The aggregator is a new feature of kubernetes and is an alpha API
+running as a separate pod in kubernetes v1.6. The aggregator is
+enabled by default in kubernetes v1.7 as a beta API, and is the
+default secure endpoint directly integrated into the core kubernetes
+APIServer.
+
+For background on why we're messing with certificates and CA's at all,
+please check [the auth doc](auth.md).
+
+# Steps
+
+## Prerequisites
+You need `cfssl` tools, install with `go get -u github.com/cloudflare/cfssl/cmd/...`.
+
+## Check that aggregator is enabled
+
+`kubectl api-versions` MUST list `apiregistration.k8s.io/v1beta1` (or
+`apiregistration.k8s.io/v1alpha1` if running in kubernetes v1.6)
+
+This won't show up if you are talking to the insecure port. The
+insecure port is not behind the aggregator.
+
+## Useful variable to define
+
+These will be used during the steps and for the final `helm
+install`. They are key to set, as the signed certificates will be
+bound to the exact service name that is defined below as
+`SVCCAT_SERVICE_NAME`.
+
+```
+export HELM_NAME=catalog
+export SVCCAT_NAMESPACE=catalog
+export SVCCAT_SERVICE_NAME=${HELM_NAME}-${SVCCAT_NAMESPACE}-apiserver
+```
+ 
+## Set up the certificate bundle
+
+The APIService expects a certificate bundle. We can create our own, or
+pull the one from kube core for reuse.
+
+The certificate bundle is made up of Certificate Authority, a Serving
+Certificate, and the Serving Private Key.
+
+### Create our own new CA and generate keys
+
+This is an example. It is written with zero understanding of the best
```

Comment:
That should not block this PR being merged.  I have no clue about best practices for SSL certs and this moves things in the right direction.